types/ramda/index.d.ts

```diff
@@ -540,246 +539,37 @@
 /// <reference path="./src/zipWith.d.ts" />
 /// <reference path="./src/includes.d.ts" />
 
-import { A, F, T, O } from "ts-toolbelt";
+import { F, T, O } from "ts-toolbelt";
+import {
+    Arity0Fn,
+    Arity1Fn,
+    Arity2Fn,
+    AssocPartialOne,
+    ComposeWithFns,
+    Dictionary,
+    Evolvable,
+    Evolve,
+    Evolver,
+    Filter,
+    Functor,
+    KeyValuePair,
+    Lens,
+    Merge,
+    MergeAll,
+    ObjPred,
+    Ord,
+    Path,
+    Placeholder,
+    Pred,
+    PipeWithFns,
+    Reduced,
+    SafePred,
+    ValueOfRecord,
+} from './tools';
 
 declare let R: R.Static;
 
 declare namespace R {
-    import ValueOfRecord = Tools.ValueOfRecord;
-    type Omit<T, K extends string> = Pick<T, Exclude<keyof T, K>>;
-
-    type Ord = number | string | boolean | Date;
```

Comment:
Will removing these from `R` break anyone using `R.Ord`, etc.?

Reply:
Hi Ron, thanks for pointing this out. I will look for an alternative solution.